modules/planning/scenarios/park/pull_over_emergency/BUILD

```diff
@@ -6,9 +6,11 @@ cc_library(
     name = "pull_over_emergency_scenario",
     srcs = [
         "pull_over_emergency_scenario.cc",
+        "stage_approach.cc",
```

Comment:
Prefer adding its own BUILD target.

Reply:
there's circular dependency among the scenario and stages. 
will build them together as is.